state-chain/pallets/cf-trading-strategy/src/tests.rs

```diff
@@ -15,18 +15,20 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use cf_primitives::{Asset, AssetAmount, Tick};
-use cf_test_utilities::assert_event_sequence;
+use cf_test_utilities::{assert_event_sequence, assert_events_eq};
 use cf_traits::{
 	mocks::{
 		balance_api::{MockBalance, MockLpRegistration},
 		pool_api::{MockLimitOrder, MockPoolApi},
 	},
 	BalanceApi, Side,
 };
-use frame_support::{assert_err, assert_ok};
+use frame_support::{assert_err, assert_noop, assert_ok, sp_runtime};
 
 use crate::{mock::*, *};
 
+pub const ALICE: <Test as frame_system::Config>::AccountId = 123u64;
```

Comment:
123 is already taken by `LP` 😄 (see mock.rs). You can either just use `LP`, or use a different id here in case you explicitly don't want them to have any role. Also, this is only used in a single place, so I would just move it there instead of giving it a wide scope (and it probably doesn't need `pub`).